postgres/content.md

```diff
@@ -184,9 +184,9 @@ Also of note, Alpine-based variants do *not* support locales; see ["Character se
 
 ## Additional Extensions
 
-When using the default (Debian-based) variants, installing additional extensions (such as PostGIS) should be as simple as installing the relevant packages (see [github.com/postgis/docker-postgis](https://github.com/postgis/docker-postgis/blob/4eb614133d6aa87bfc5c952d24b7eb1f499e5c7c/12-3.0/Dockerfile) for a concrete example).
+When using the default (Debian-based) variants, installing additional extensions (such as PostGIS) should be as simple as installing the relevant packages (see [github.com/postgis/docker-postgis](https://github.com/postgis/docker-postgis/blob/master/14-3.2/Dockerfile) for a concrete example).
```

Comment:
```suggestion
When using the default (Debian-based) variants, installing additional extensions (such as PostGIS) should be as simple as installing the relevant packages (see [github.com/postgis/docker-postgis](https://github.com/postgis/docker-postgis/blob/c69d7b253205835558d8048430c71588cae0a593/14-3.2/Dockerfile) for a concrete example).
```

These should be a specific commit so that when master changes and loses the `14-3.2` directory, it still works (a working outdated link is better than a 404). Similar change needed for the other postgis link.

Reply:
Done, see https://github.com/docker-library/docs/pull/2161#issuecomment-1152673357